internal/commands/app/create_inputs.go

```diff
@@ -131,6 +135,65 @@ func (i *createInputs) resolveLocalPath(ui terminal.UI, wd string) (string, erro
 	return fullPath, nil
 }
 
+func (i *createInputs) resolveTemplateID(ui terminal.UI, client realm.Client, templateID string) (string, error) {
+	if templateID == "" && ui.AutoConfirm() {
```

Comment:
yeah, but then how would you handle the case where a user is using `-y` (perhaps already, in a CI flow or something) without breaking things for them when we introduce this feature? if the requirement is that we always prompt a user for a template option _unless_ they've specified a `--template` param, we want to be able to skip the prompt/required param for those users using `-y`. otherwise, we either force them to provide a `--template none` (which doesn't feel right).